lib/cli.ex

```diff
@@ -1,7 +1,20 @@
 defmodule Onigumo.CLI do
-  def main([component]) do
-    module = Module.safe_concat("Onigumo", component)
+  def main(argv) do
+    {_parsed, args, _invalid} = OptionParser.parse(argv, strict: [])
```

Comment:
If we match the invalid arguments with `[]`, we‘ll get a MatchError. Like that, we can leverage the current [rescue clause](https://github.com/Glutexo/onigumo/blob/f7272069fcc32abdc4ddc6333fae150d657c0f08/lib/cli.ex#L10C5-L10C11) catching for _OptionParser.ParseError_ and _MatchError_. It’s not something we want to keep as we want to abandon raising exceptions (see #204), but it can be a nice step that keeps a minimal amount of changes.

The same applies for the parsed argument, which we [already do](https://github.com/Glutexo/onigumo/blob/f7272069fcc32abdc4ddc6333fae150d657c0f08/lib/cli.ex#L8) in the codebase. They both would raise a _MatchError_ and we currently don’t need to differentiate between them.

```suggestion
    {[], args, []} = OptionParser.parse(argv, strict: [])
```